jetstream/consumer.ts

```diff
@@ -139,7 +139,7 @@ export type Expires = {
 
 export type Bind = {
   /**
-   * If set to false the client will not try to check on its consumer by issuing consumer info
+   * If set to true the client will not try to check on its consumer by issuing consumer info
    * requests. This means that the client may not report consumer not found, etc., and will simply
```

Comment:
would it help to add a sentence about consumer info requests potentially being expensive on the server?

Reply:
Not sure that is appropriate - specially since that is a server issue that should be addressed there.